### ISSUE TITLE
Instant Jam Button

### ISSUE THREAD

**Issue by Nikita74RU** [BEEmod/BEE2-items#2665]
Weighted or Cube or Ball. Will have orange active skin, when jammed with a cube, ball, or player will become blue unactive skin, you can get off anything on the button, but it still jammed in. And sends output.

**Comment by King0191** [comment]
I think i have seen a button like this. so, if i may, I will revise this. This button looks like a normal button. But its a blue (Best choose a different shade of blue than the Portal ON/OFF button.) button. No cubes can activate the button. But when you step on it. It works. When you step off of the button it stays lowered. Causing the output to stay active. This would be interesting if added. Due to what you can create with this button in mind. However this is just a suggestion. And probably better if i made a different issue for it (I might still. We'll see what happens)

**Comment by King0191** [comment]
And you really aren't wrong. However a Button takes 7 entities (how? idk) and Buttons take 2 entities, an SR Latch takes 3 entities and, while im not one of them, Some people are REALLY stingy with their entity costs. It would also make things slightly easier for the Map Maker to create. Just place one item and connect it. Rather than 4 items and have to connect all of them to something else. Im not saying that its the best alternative to those things you've mentioned But im saying that it could help in some cases. Like a... wow ok.. Im gonna say something really lame but a lot of people make them. Companion Cube Stages. One Companion Cube. Whole map. perhaps the exit door is in the middle. and half way across the map is a button for the cube. And on the other side, theres another button. but no portal surfaces. That test is impossible. So solution? Portable surfaces wouldn't help. you'd still deactivate the door. So you could use this one-use button to solve this test with ease. Maybe add a portal surface so they don't have to walk back. Or open a shortcut. Either one works. And like i said Hugo. This doesn't have to be added. And i wont go into deep depression if it isn't. I just think it would be an interesting concept.

**Comment by HugoBDesigner** [comment]
Well, like I mentioned, seems oddly case-specific

**Comment by King0191** [comment]
Yeah. To be honest. I don't think it would have a ton of cases where it would be used. But i brought it up due to Nikita bringing up something similar. But then again,  Case-specific items aren't new to BEEMod, Many of the fizzlers can only be used in specific situations where their properties would work. But hey. Im not a developer. You guys know much more then i do about this.

**Comment by LautaroL20** [comment]
@TeamSpen210 - Do not forget to apply the label after closing an issue; since although it may sounds unbelievable, there are some people who still checks inside the "closed issues" tab in order to see which issues were solved/closed and their reasons. So, having them shorted with the proper label after closing will help people to understand the real cause of its closure.

Do not apply aditional labels such as "enhancement", "new item" etc, when the issue is invalid/duplicate. The "invalid" or "duplicate" labels are just enough to clarify a reason for it.
